### PR TITLE
Feat/add oif summit 2023 proposal open collaboration

### PR DESCRIPTION
**Pull request by fkr**

### CODE REVIEW

File: OpenInfraSummit2023/Open-Collaboration-viewed-through-the-eyes-of-a-PO.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+Track: Open Development
+
+Authors: Felix Kronlage-Dammers
+
+Technical Level: Beginner
+
+## Abstract (1000 chars)
+
+The Sovereign Cloud Stack (SCS) orchestrates various standardization and development
+efforts in an open and collobaritve fashion. The projects development efforts typically
```
Comment (itrich) [inline OpenInfraSummit2023/Open-Collaboration-viewed-through-the-eyes-of-a-PO.md]: *collaborative*

Comment (itrich) [inline OpenInfraSummit2023/Open-Collaboration-viewed-through-the-eyes-of-a-PO.md]: The development typically happens in the ...

File: OpenInfraSummit2023/Open-Collaboration-viewed-through-the-eyes-of-a-PO.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+The Sovereign Cloud Stack (SCS) orchestrates various standardization and development
+efforts in an open and collobaritve fashion. The projects development efforts typically 
+happen in the corresponding upstream project - done through funding and tender projects
+just as much as through interested parties at the various cloud service providers that
+utilize the SCS stack. Coordination happens in recurring meetings facilitated by the
+product owners at the SCS project - embracing the four opens!
+In the course of this talk the Product Owner for the IaaS Reference Implementation of the
+Sovereign Cloud Stack, which is based on OpenStack obviously, will dive into how the
+development process works and will share insights on what went well in the first years
+and where things needed (and still need) improvement and how these hurdles are tackled.
```
Comment (itrich) [inline OpenInfraSummit2023/Open-Collaboration-viewed-through-the-eyes-of-a-PO.md:20]: I suggest to split this rather long sentence.

> In the course of this talk the Product Owner for the IaaS Reference Implementation of the
Sovereign Cloud Stack, which is based on OpenStack obviously, will dive into how the
development process is managed. Felix will share insights on what went well in the first years, where things needed (and still need) improvement and how these hurdles are tackled.

File: OpenInfraSummit2023/Open-Collaboration-viewed-through-the-eyes-of-a-PO.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+Focus will be placed on the facilitation of these, very special, sprint plannings and reviews
```
Comment (itrich) [inline OpenInfraSummit2023/Open-Collaboration-viewed-through-the-eyes-of-a-PO.md]: The focus of this talk will be placed on...

(Grammarly suggests to always start with an article) :smile:

**Review by itrich — CHANGES_REQUESTED** [review]
I'm already looking forward to learning from you! :pray:

**Comment by fkr** [comment]
@itrich thanks for the feedback. Incorporated your changes, please re-review.

File: OpenInfraSummit2023/Open-Collaboration-viewed-through-the-eyes-of-a-PO.md (hunk 1 above)
Comment (itrich) [reply OpenInfraSummit2023/Open-Collaboration-viewed-through-the-eyes-of-a-PO.md]: collaborative is still misspelled

**Review by itrich — DISMISSED** [review]
collaborative is still misspelled. Besides that: LGTM!

**Comment by fkr** [comment]
> collaborative is still misspelled. Besides that: LGTM!

I'm such a n00b. thanks ;)

**Review by itrich — APPROVED** [review]
LGTM

**Pull request merged**